README.md

```diff
@@ -25,20 +25,21 @@ Mark Nottingham has a [nice blog]( http://www.mnot.net/blog/2012/09/05/patch) ab
 
 ## Performance
 
-##### [`add` benchmark](https://run.perf.zone/view/JSON-Patch-Add-Operation-1500650502896)
+##### [`add` benchmark](https://run.perf.zone/view/JSON-Patch-Add-Operation-1527975263015)
 
-![image](https://user-images.githubusercontent.com/17054134/28513264-b0c8eede-7055-11e7-8204-4536d871e971.png)
+![image](https://user-images.githubusercontent.com/17054134/40881094-d8a5f662-66be-11e8-88a8-f00a7117076e.png)
 
-##### [`replace` benchmark](https://run.perf.zone/view/JSON-Patch-Replace-Operation-1500645548223)
+##### [`replace` benchmark](https://run.perf.zone/view/JSON-Patch-Replace-Operation-1527975551191)
 
-![image](https://user-images.githubusercontent.com/17054134/28513298-d8c927a0-7055-11e7-9a4e-2da024655ac0.png)
+![image](https://user-images.githubusercontent.com/17054134/40881090-b0d21df0-66be-11e8-9279-b35ba65f2e1e.png)
```

Comment:
In the legend for first 4 libs you are using author, but for the 5th you used package name. Is there a reason to do so? I'd keep it consistent and use either name or author in all.